GitHubReactions.ps1

```diff
@@ -204,47 +259,76 @@ filter Set-GitHubReaction
+            -ReactionType rocket
 
         Sets the 'rocket' reaction for issue 12626 of the PowerShell\PowerShell project.
 
     .EXAMPLE
-        Get-GitHubPullRequest -Uri https://github.com/microsoft/PowerShellForGitHub -PullRequest 193 | Set-GitHubReaction -ReactionType Heart
+        Get-GitHubPullRequest -Uri https://github.com/microsoft/PowerShellForGitHub `
+            -PullRequest 193 | Set-GitHubReaction -ReactionType Heart
 
-        Gets a GitHub pull request and pipes it into Set-GitHubReaction to set the 'heart' reaction for that pull request.
+        Gets a GitHub pull request and pipes it into Set-GitHubReaction to set the
+        'heart' reaction for that pull request.
 
     .NOTES
-        Currently, this only supports reacting to issues and pull requests. Issue comments, commit comments and PR comments will come later.
+        Currently, this only supports reacting to issues and pull requests.
+        Issue comments, commit comments and PR comments will come later.
 #>
     [CmdletBinding(
         SupportsShouldProcess,
-        DefaultParameterSetName='Elements')]
+        DefaultParameterSetName='ElementsIssue')]
     [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification="Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
     [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSReviewUnusedParameter", "", Justification="One or more parameters (like NoStatus) are only referenced by helper methods which get access to it from the stack via Get-Variable -Scope 1.")]
     param(
         [Parameter(
             Mandatory,
-            ParameterSetName='Elements')]
+            ParameterSetName='ElementsIssue')]
+        [Parameter(
+            Mandatory,
+            ParameterSetName='ElementsPullRequest')]
         [string] $OwnerName,
 
         [Parameter(
             Mandatory,
-            ParameterSetName='Elements')]
+            ParameterSetName='ElementsIssue')]
+        [Parameter(
+            Mandatory,
+            ParameterSetName='ElementsPullRequest')]
         [string] $RepositoryName,
 
         [Parameter(
             Mandatory,
             ValueFromPipelineByPropertyName,
-            ParameterSetName='Uri')]
+            ParameterSetName='UriIssue')]
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName='UriPullRequest')]
         [Alias('RepositoryUrl')]
         [string] $Uri,
 
-        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
-        [Parameter(Mandatory, ParameterSetName='Uri', ValueFromPipelineByPropertyName)]
-        [Alias('Issue')]
-        [Alias('PullRequest')]
+        [Parameter(
+            Mandatory,
+            ParameterSetName='ElementsIssue',
+            ValueFromPipelineByPropertyName)]
```

Comment:
Please keep the `ParameterSetName` the last entry, as with the other examples throughout the module.  Here and elsewhere.